platform/pom.xml

```diff
@@ -29,17 +29,11 @@
     <packaging>pom</packaging>
 
     <properties>
-        <cxf.version>3.0.4</cxf.version>
         <cxf.jaxb.version>2.2.6</cxf.jaxb.version>
-        <commons-io.version>2.1</commons-io.version>
         <joda-time.version>2.2</joda-time.version>
         <joda-convert.version>1.7</joda-convert.version>
-        <org.slf4j.version>1.7.1</org.slf4j.version>
         <osgi.version>5.0.0</osgi.version>
```

Comment:
The osgi version property is define by the karaf parent pom which is imported.  We should be able to remove this property.